src/db.rs

```diff
@@ -179,6 +182,42 @@ pub async fn run_migrations(client: &DbClient) -> anyhow::Result<()> {
     Ok(())
 }
 
+pub async fn run_scheduled_jobs(db: &DbClient) -> anyhow::Result<()> {
+    let jobs = get_jobs_to_execute(&db).await.unwrap();
+    println!("jobs to execute: {:#?}", jobs);
+    tracing::trace!("jobs to execute: {:#?}", jobs);
+
+    for job in jobs.iter() {
+        update_job_executed_at(&db, &job.id).await?;
+
+        match handle_job(&job.name, &job.metadata).await {
```

Comment:
How do you think we should run each handler async?

We tried spawning a [task](https://docs.rs/tokio/latest/tokio/task/) for each but there are some challenges propagating the errors (handling Result), so we are open to any suggestion here 😄

Reply:
This is fine. We shouldn't really have long running tasks, so awaiting inline seems fine to me.